Sources/HTTP/HTTPVersion.swift

```diff
@@ -20,6 +20,17 @@ public struct HTTPVersion {
     }
 }
 
+extension HTTPVersion {
+    /// HTTP version 1.0
+    public static let v1_0 = HTTPVersion(major: 1, minor: 0)
```

Comment:
I don't know, maybe, why not. I'm not sure when I would ever use those constants.

Reply:
Maybe someone has an old copy of Netscape lying around, and can't use HTTP 1.1. :(

Reply:
HTTP 1.0 and 1.1 are different only in default values for `Keep-Alive` and `Connection` headers afaik. We will replace instances in current code like [here](https://github.com/swift-server/http/blob/31d066a0e92b4575a4389d592e022fe0b9933956/Sources/HTTP/HTTPStreamingParser.swift#L401)